portalnet/src/overlay_service.rs

```diff
@@ -82,16 +83,10 @@ const EXPECTED_NON_EMPTY_BUCKETS: usize = 17;
 /// Bucket refresh lookup interval in seconds
 const BUCKET_REFRESH_INTERVAL_SECS: u64 = 60;
 
-/// The default configuration to use for uTP connections.
-pub const UTP_CONN_CFG: ConnectionConfig = ConnectionConfig {
-    max_packet_size: 1024,
-    max_conn_attempts: 3,
-    max_idle_timeout: Duration::from_secs(32),
-    max_timeout: Duration::from_secs(60),
-    initial_timeout: Duration::from_millis(1500),
-    min_timeout: Duration::from_millis(500),
-    target_delay: Duration::from_millis(250),
-};
+lazy_static! {
+    /// The default configuration to use for uTP connections.
+    pub static ref UTP_CONN_CFG: ConnectionConfig = ConnectionConfig { max_packet_size: 1024, ..Default::default()};
```

Comment:
@carver You might want to read-through / follow-up on the conversation that happened [here](https://github.com/ethereum/trin/pull/767) about an accurate max packet size. It's been a while since I was thinking about this, but it seems like we may be able to fit a few more bytes in here. Don't consider this necessary for this pr, but tracking it in an issue/comment would be nice.

Reply:
Tracked in #910